### ISSUE TITLE
[Request] either impliment or add an upgrade that can fill Automined ore blocks with a cheep filler (Cobblestone or something)

### ISSUE THREAD

**Issue by SteamDoge9000** [GTNewHorizons/GT-New-Horizons-Modpack#8422]
This is self explanatory and would probably not be a bad idea to add because if a "Gap" is big enough, it can spawn mobs and that can result in more lag or even infernals spawning inside hidden crevices that would make it a PAIN to try and locate

if you go "add it to the machine" route then maybe a modified frame or something that involves a cobblestone generator or some sort could be used

or 

if you go the "Upgrade" route, you can install the upgrade to act as a independent buffer that will pull valid items (cobble n stuff) from inventories facing it; can be removed with crowbar

**Comment by SteamDoge9000** [comment]
![image](https://user-images.githubusercontent.com/38770780/129811393-c1e53fbc-5701-4f34-a406-a2622a744de4.png)

not a bad idea, maybe a config could be made for it to it to register what blocks to use if its in X dimension

**Comment by bombcar** [comment]
All gregtech ores spawn in only very specific types of blocks/rock - of which most have a cobble version available. It makes sense for the MBM and above to crush the ore in place - extract the minerals - and leave the cobble behind. 

An ability to do this with the normal miners would also be nice even if you have to feed it somehow

**Comment by Prometheus0000** [comment]
That's not true though? Otherwise there would be nothing spawning on other planets. Also, giving the ability to the normal ones is dumb, just use the concrete backfiller.

PS: it should use liquid concrete to fill in the holes, be a different mode, and use more power if you want it to be balanced. Or not, if we don't care.

**Comment by GTNH-Colen** [comment]
Right, because so many people actually use that multiblock. This is not about making some nonsense argument for realism, but one can be made if you must justify it that way. I really don't see what purpose making thousands of empty holes underground for mobs to spawn does? It makes perfect sense to fill them back in with cobblestone and prevent a lag source. We all know damn well that no one is going to bother going back over an area with a concrete backfiller unless forced.

What are the actual downsides? What are we scared of happening? People mining FREE cobblestone? The absolute horror of it. Why overcomplicate it with new miner modes and whatnot when we can probably just change a few lines of code and throw cobblestone in? It's not like anyone actually cares if there is a few blocks of cobble under some random planet surface they're never going to visit again, but if someone wants to texture new cobble blocks no one will ever see then feel free.

**Comment by Prometheus0000** [comment]
It's not like I'm seriously suggesting it. Though adding the ability to the SB one feels like it makes it have too many features for a SB, I understand that lag is more important.

**Comment by ShyZShark** [comment]
Admittedly this is sort of petty, I realize that so please don't be too harsh, but if we're going on the route of lag prevention / optimization, then I'd suggest changing this to standard Stone / default filler world blocks for that dim. From my experience, doing that with other packs using the RFTools Builder, due to how Minecraft saves the world region files, if I understand that correctly as well, it would reduce world sizes and improve server to client chunk loading times. I've done that many times in other packs and for those that don't have stellar internet like myself, it would improve their experience on servers, even if in a minor but noticeable way. It also makes the world look a little more clean, especially for surface spawning small ores.

Only downside would be that other players can't tell with extreme ease that an auto miner has ran in that area, but I'd argue that if you aren't running into small ores in the first place, that players will just assume the vein didn't spawn, which naturally happens anyways. I suppose it also lowers "realism" which is merely an implied illusion in this pack, as it is in basically every game ever made, so that doesn't seem like a large concern, at least to me. Not over the theoretical improvements.
